src/main/java/com/adyen/serializer/SaleToAcquirerDataModelAdapterSerializer.java

```diff
@@ -0,0 +1,16 @@
+package com.adyen.serializer;
+
+import com.adyen.model.terminal.SaleToAcquirerDataModel;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonPrimitive;
+import com.google.gson.JsonSerializationContext;
+import com.google.gson.JsonSerializer;
+
+import java.lang.reflect.Type;
+
+public class SaleToAcquirerDataModelAdapterSerializer implements JsonSerializer<SaleToAcquirerDataModel> {
```

Comment:
Any specific reason to mention Adapter in name of class ?

Reply:
not particular I will remove the adapter part.